### ISSUE TITLE
Make width of the sttings bar adjustable

### ISSUE THREAD

**Issue by Fass-ohne-Hoden** [Ultimaker/Cura#119]
The new Cura has settings bar on the right side, with a fixed width. In some languages, a lot of the settings captions are shortened to unreadability.

Make the width of that settings bar adjustable (in a grab-the-border-and-drag-it-around kind of way).

**Comment by nallath** [comment]
Would changing a preference setting be sufficient?

**Comment by Fass-ohne-Hoden** [comment]
You mean, like, inputting the width in pixels?
... maybe not the most intuitive option, but it would work, sure. In that case, could it be made so that the setting takes effect instantly (at least, without restarting the program)? Although, one is likely to configure this only once or twice, it'd be a hassle to iterate through that thing (change value; restart; check [crap, too much]; changing value again, restart ...)

**Comment by nallath** [comment]
Yeah I agree, but I'm not quite sure how hard it would be to make it dragable. Hence my question.

**Comment by awhiemstra** [comment]
Draggable should be doable, but I would rather make sure the default width of the sidebar is correct. Of course, that is quite a bit harder to do.

The primary reason for this is that I want to make it possible to put the sidebar in a two or more column mode so you can have a better overview of all the settings available on larger screens. This would be someting I think makes sense to have attached to a draggable edge of the sidebar.

**Comment by BagelOrb** [comment]
I think this feature is also nice to have when you have a small screen. 

Putting the settings into two (or more) bars wouldn't necessarily create more overview. You should then always cut off the settings at [Travel], instead of an adjustable cut. That way the placement of the settings is more predictable.

I think this issue is low priority, though.

**Comment by nallath** [comment]
Layout will be changed, see https://github.com/Ultimaker/Cura/issues/194
